src/main/tools/linux-sandbox-options.cc

```diff
@@ -263,6 +263,9 @@ void ParseOptions(int argc, char *argv[]) {
 
   if (opt.working_dir.empty()) {
     char *working_dir = getcwd(nullptr, 0);
+    if(!working_dir) {
```

Comment:
Could you check against nullptr here?

Reply:
You mean, `working_dir != nullptr`? May I ask why?

Reply:
Since it's not a boolean -- passing along the (nit) comment from internal review. I changed it there, though, so no action needed.